packages/next/backend/scripts/spaces/interlay.js

```diff
@@ -0,0 +1,20 @@
+const config = {
+  id: "interlay",
+  name: "Interlay",
+  symbol: "INTR",
+  decimals: 10,
+  networks: [
+    {
+      network: "interlay",
+      ss58Format: 2032,
+    },
+  ],
+  proposeThreshold: "1000000000000",
+  voteThreshold: "10000000000",
```

Comment:
I think we can lower the threshold for proposer to `1 INTR`, so it will be `1 * Math.pow(10, 10) = 10000000000`. It's same for vote threshold, and we can remove  2 '0'.

Reply:
Make sense to me, will apply the change.